sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,38 +1,37 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (20/11/10)
 
 ### Breaking Changes
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
 - Removed `azure.hpp`.
 
+### Bug Fixes
+
+- Prevent pipeline of length zero to be created.
+- Avoid re-using a connection when a request to upload data fails while using the `CurlTransport`.
+
 ### New Features
 - Added `strings.hpp` with `Azure::Core::Strings::LocaleInvariantCaseInsensitiveEqual` and `Azure::Core::Strings::ToLower`.
 - Added `OperationCanceledException`.
 - Added `GetPort()` to `Url`.
 - Added `TransportPolicyOptions`.
 - Added `TelemetryPolicyOptions`.
 - Added `DateFormat::ToIso8601String(TimeFractionFormat)`.
+- Added `RequestFailException` deriving from `std::runtime_error`.
+- Added `CurlTransportOptions` for the `CurlTransport`.
+- Added `DateTime` supporting dates since 1601.
 
 ### Other changes and Improvements
 
 - Add high-level and simplified core.hpp file for simpler include experience for customers.
 - Add code coverage using gcov with gcc.
 - Update SDK-defined exception types to be classes instead of structs.
```

Comment:
Make libcurl network requests cancelable by Context::Cancel()
https://github.com/Azure/azure-sdk-for-cpp/pull/719

Reply:
Also add:
https://github.com/Azure/azure-sdk-for-cpp/pull/807

@vhvb1989 - please go through and review any other relevant customer facing changes we need to add to the changlog.

Reply:
#719 -> it is below `Make it possible to cancel a request while writing or reading from the network.`

I will use yours if you want

Reply:
>@vhvb1989 - please go through and review any other relevant customer facing changes we need to add to the changelog.

I dit it. Gotta trust me a little bit XD

Reply:
> I will use yours if you want

No, yours is good. I missed it.